server/src/auth/index.ts

```diff
@@ -10,6 +13,30 @@ import { getDb } from '../database';
 import { configureAzureAuth } from './azure';
 import { configureGoogleOAuth } from './google-oauth';
 
+function basicAuth(req: Request): { name: string; pass: string } {
+  const string = req?.headers?.authorization;
+  if (!string) return;
+
+  const match = /^ *(?:[Bb][Aa][Ss][Ii][Cc]) +([A-Za-z0-9._~+/-]+=*) *$/.exec(
```

Comment:
[nit] You can make regexes case-insensitive with an `i` flag at the end of it:
```suggestion
  const match = /^ *(?:Basic) +([A-Z0-9._~+/-]+=*) *$/i.exec(
```